core-bundle/contao/templates/backend/be_two_factor.html5

```diff
@@ -133,15 +133,15 @@
             </tr>
           </thead>
           <tbody>
-            <?php foreach ($this->credentials as $credential): ?>
+            <?php foreach ($this->credentials as $i => $credential): ?>
               <tr class="hover-row">
                 <td class="name">
                   <?php if ($this->editPassKeyId === $credential->getId()): ?>
                     <form method="post" data-contao--passkeys-target="form" id="form-edit-passkey-<?= $credential->getId() ?>">
                       <input type="hidden" name="FORM_SUBMIT" value="tl_passkeys_credentials_edit">
                       <input type="hidden" name="REQUEST_TOKEN" value="<?= $this->requestToken ?>">
                       <input type="hidden" name="credential_id" value="<?= $credential->getId() ?>">
-                      <input type="text" name="passkey_name" value="<?= Contao\StringUtil::specialchars($credential->name) ?>" placeholder="<?= $this->trans('MSC.passkeyNameInput') ?>" data-action="keydown.esc->contao--passkeys#cancelEdit" data-contao--passkeys-target="nameInput">
+                      <input type="text" name="passkey_name" value="<?= Contao\StringUtil::specialchars($credential->name) ?>" placeholder="<?= $this->trans('MSC.passkeyNameInput') ?>" aria-label="<?= Contao\StringUtil::specialchars($this->trans('MSC.passkeyNameInput')) ?>" data-action="keydown.esc->contao--passkeys#cancelEdit" data-contao--passkeys-target="nameInput">
```

Comment:
Why this? And why is the first one not wrapped in `specialchars()` and the second one is?